modular_skyrat/modules/aesthetics/airlock/code/airlock.dm

```diff
@@ -1,10 +1,12 @@
 //SKYRAT ADDITION BEGIN - AESTHETICS
 #define AIRLOCK_LIGHT_POWER 1
 #define AIRLOCK_LIGHT_RANGE 2
+#define AIRLOCK_LIGHT_ENGINEERING "engineering"
 #define AIRLOCK_POWERON_LIGHT_COLOR "#3aa7c2"
 #define AIRLOCK_BOLTS_LIGHT_COLOR "#c23b23"
 #define AIRLOCK_ACCESS_LIGHT_COLOR "#57e69c"
-#define AIRLOCK_EMERGENCY_LIGHT_COLOR "#d1d11d"
+#define AIRLOCK_EMERGENCY_LIGHT_COLOR "#57e69c"
+#define AIRLOCK_ENGINEERING_LIGHT_COLOR "#ffaece"
```

Comment:
![image](https://user-images.githubusercontent.com/76465278/195329965-6927806f-e022-419c-91fa-b2f34219d6af.png)

![image](https://user-images.githubusercontent.com/76465278/195330084-2f808e6f-0973-49d2-9a10-dfb44cb62127.png)

~Please tell me these look better in game lmaooo~ 

![image](https://user-images.githubusercontent.com/76465278/195331893-14170ed7-feca-4bd0-ae86-ba6d8173f76b.png)

Found the sprites that don't use this (does ANYTHING use this color? Is it the lighting effect behind the door?)
I must say they... really don't fit. The pink on most doors is too close to Denied red, and the green just looks out of place. Maybe they look better on the door, but I'd be surprised if so

Why not leave the emergency light color yellow, and have engineering emergency lights orange? Setting a door to emergency doesn't make it an Exit, it makes it _All-Access_; green light doesn't portray `This area is in a unique, potentially urgent or threatening condition` like yellow does - it screams `safety` which is rarely the actual use of emergency access. Usually it's the opposite.
Plus, green is already used for opening doors.

And... pink? Is there a reason for that choice?

Reply:
I'll be real, green makes more sense to tell me that "Hey I can open this door" vs one that in some cases blends in with the door color itself, the crew are trained to know what the alert is anyway. I'm not 100% but Pink's a standard alert color along with Green and Orange as it's a visibility deal

Reply:
According to the MUTCD, trying to reflect real world, pink is to be used for “incident management.” Same reason for changing the emergency access to the international green.

I'll don't know why it thinks sparks changed there, I'll take a look and remove the nameless one.

Reply:
Having flashing yellow and flashing orange as two different door states would also be too similar.

Reply:
The change to firelock.dmi changes the delay between frames from 10 to 12 so they're in sync with the flashing airlocks.

Reply:
Pink incident management checks out, but make sure its not too similar to Bolted red; high security airlock specifically looks really similar
Per your source though, green is for Guidance markers; yellow, for Warnings, is still much more fitting to Emergency Access. You can try to tweak it a tad to not blend as much though, thats a valid point

Reply:
It doesn't clash with bolted red.
![image](https://user-images.githubusercontent.com/83487515/195477601-8552b17b-7dba-49bc-8faa-273040f0bf6d.png)

Reply:
AIRLOCK_ENGINEERING_LIGHT_COLOR is the light generated by the airlock, and #ffaece is a much softer light, because for this matching the airlock light seemed too harsh of a light.

This is matching the emissive to the airlock color
![image](https://user-images.githubusercontent.com/83487515/195478584-60380666-3001-4f8c-9d9f-e85eb3f23d83.png)

Reply:
Yea the pink checks out then, just make sure to double check on the reinforced(Vault) one too, its really the only one I was worried about

Reply:
Yea the pink checks out then, just make sure to double check on the reinforced(Vault) one too, its really the only one I was worried about

Reply:
Pink might be what's used in real life situations, but like... Honestly, it feels a little wrong to me in a SS13 sense, I think it'd be better off having it be orange, to stay in theme with what players already expect. It being pink will mean that most players will have to learn from an outside source that "hey this is emergency engineering access" whereas orange is pretty straight-forward in that sense.

Reply:
On the other hand, a bunch of people could learn an interesting real world thing from the spess game. I think people would understand the pink lights rather quickly in practice.